### PR TITLE
Update assistance flow to also save current submission as draft

### PR DESCRIPTION
**Pull request by wilwong89**
# Description
When using the assistance flow tab, it will save any fields filled in for submission as a draft.
It will then create an enquiry with the filled in applicant information with an "Assistance" status.
The Enquiry and draft Submission will be connected through their activity IDs.

[PADS-253](https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-253)

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [X] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Subin1Doo** [comment]
Everything seems to be working fine! Just one thing, after I submit the assistance and go look at the saved draft, the missing fields are highlighted in red. Can we fix that. Thank you!

**Comment by wilwong89** [comment]
> Everything seems to be working fine! Just one thing, after I submit the assistance and go look at the saved draft, the missing fields are highlighted in red. Can we fix that. Thank you!

Unfortunately not, the library for validation library triggers that and does not provide a way to bypass.

**Review by kyle1morel — APPROVED** [review]

File: frontend/src/components/housing/submission/SubmissionAssistance.vue [hunk 1]
```diff
@@ -46,44 +43,19 @@ const checkApplicantValuesValid = (
   return isValid;
 };
 
-const confirmSubmit = (data: any) => {
+const confirmSubmit = () => {
   confirm.require({
-    message: 'Are you sure you wish to submit this form? Please review the form before submitting.',
-    header: 'Please confirm submission',
+    message: 'Are you sure you wish for assistance with this form? Please review the form before submitting.',
```
Comment (jujaga) [inline frontend/src/components/housing/submission/SubmissionAssistance.vue:48]: English: Something feels off with this sentence... would be using a more active/direct tense to say something along the lines of
```txt
Are you sure about requesting assistance with this form? Please review the form before submitting.
```

Comment (wilwong89) [reply frontend/src/components/housing/submission/SubmissionAssistance.vue:48]: Consulting with Subin for better wording now. Thanks for pointing it out.

Comment (Subin1Doo) [reply frontend/src/components/housing/submission/SubmissionAssistance.vue:48]: totally missed it, it does sound awkward.. I like what you suggested, or .. we can do something like "Are you sure you wish to submit this form for assistance? Please review the form before submitting."

**Review by sanjaytkbabu — APPROVED** [review]

**Pull request merged**